### PR TITLE
Fixes home screen banner issue

### LINKED ISSUE #911: Fix aspect ratio of main banner in home screen
**Is your feature request related to a problem? Please describe.**

when the device is rotated, the main Kiwix banner looks distorted.

Here's the screen:
screenshot_20190109-023355

And heres how it looks when the device is rotated:
screenshot_20190109-024049

**Describe the solution you'd like**

we could create the banner using three layers, (one for the main rectangle base, one for the kiwix logo, and one for the side pattern), instead of one. Then, in rotated mode, we can just change the size of the base layer, and change the relative positions of the side pattern, and the kiwix logo).

### PR DESCRIPTION
**Pull request by sonusourav**
Fixes #911 

Changes: Used different kiwix logo design to make up for the space in landscape orientation but the main design is kept intact. 

Screenshots/GIF for the change: 
banner fix

### CODE REVIEW

**Comment by sonusourav** [comment]
@abdulwd @mhutti1 I have opened a new pull request because I messed up doing squash. Hope you will consider this and I have closed the previous one.

File: app/src/main/res/layout-land/content_main.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (abdulwd) [inline app/src/main/res/layout-land/content_main.xml]: This file was not there in your previous PR. Check if you can do it with one layout.

Comment (sonusourav) [reply app/src/main/res/layout-land/content_main.xml]: I have removed this file. A separate xml is not required for landscape. I could do that using drawables only. Thanks for pointing it out.

**Comment by iadeelzafar** [comment]
Nice work. This looks good.

**Comment by sonusourav** [comment]
@abdulwd Please review it.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -23,7 +23,8 @@
       android:label="@string/app_name"
       android:supportsRtl="true"
       android:theme="@style/AppTheme"
-      android:networkSecurityConfig="@xml/network_security_config">
```
Comment (mhutti1) [inline app/src/main/AndroidManifest.xml]: Why have you removed this line?

Comment (sonusourav) [reply app/src/main/AndroidManifest.xml]: @mhutti1 It was due to some merge issue. Anyway, I have resolved them now.

**Pull request closed without merging**